go.mod

```diff
@@ -176,6 +177,9 @@ require (
 )
 
 replace (
+	// // TODO tag all extracted modules after SDK refactor
```

Comment:
Is it intended?

Reply:
no, I forgot to remove these lines. Deleted now.